### PR TITLE
[1/2] Add workflow commands

### PR DESCRIPTION
**Pull request by darkLord19**
- Get info about workflow
```
/circleci workflow get <workflow id>
```
- Get jobs of given workflow
```
/circleci workflow jobs <workflow id>
```

### CODE REVIEW

File: server/command.go [hunk 1]
```diff
@@ -97,11 +97,20 @@ func getAutocompleteData() *model.AutocompleteData {
 
 	// Config
 	configCommand := commands.GetConfigAutoCompeleteData()
+	workflow := model.NewAutocompleteData(workflowTrigger, workflowHint, workflowHelpText)
+	workflowGet := model.NewAutocompleteData(workflowGetTrigger, workflowGetHint, workflowGetHelpText)
+	workflowGet.AddTextArgument("workflow id", workflowGetHint, "")
+	workflowGetJobs := model.NewAutocompleteData(workflowGetJobsTrigger, workflowGetJobsHint, workflowGetJobsTriggerHelpText)
+	workflowGetJobs.AddTextArgument("workflow id", workflowGetJobsHint, "")
+	workflow.AddCommand(workflowGet)
+	workflow.AddCommand(workflowGetJobs)
```
Comment (nathanaelhoun) [inline server/command.go]: Please add a `GetWorkflowAutocompleteData()` method in the `commands_workflow` file

**Review by nathanaelhoun — APPROVED** [review]
Didn't tested but LGTM! (small changes is needed though, checks are failing)

File: server/command.go (hunk 1 above)
Comment (darkLord19) [reply server/command.go]: done

**Comment by darkLord19** [comment]
@nathanaelhoun any idea why circle ci lint job is getting killed?
```
go: downloading github.com/mattermost/mattermost-server/v5 v5.3.2-0.20200723144633-ed34468996e6
go: downloading golang.org/x/net v0.0.0-20200625001655-4c5254603344
Checking for style guide compliance
Running golangci-lint
golangci-lint run ./...
make: *** [Makefile:46: check-style] Killed

Exited with code exit status 2
```

**Pull request merged**